src/common/MovieTile/styled.js

```diff
@@ -3,27 +3,33 @@ import { ReactComponent as YellowStar } from "../../images/shape-star.svg";
 
 export const Wrapper = styled.div`
     max-width: 324px;
-    height: 650px;
+   /*  min-height: 650px; */
```

Comment:
I don't think this comment is needed here 👌

Reply:
got me ! I forgot to deleted it :) 

already done !